packages/appkit/src/client/appkit-base-client.ts

```diff
@@ -1585,6 +1585,7 @@ export abstract class AppKitBaseClient {
     const authConnector = ConnectorController.getAuthConnector(namespace)
     const accountState = ChainController.getAccountData(namespace)
     const activeChain = ChainController.state.activeChain as ChainNamespace
+    const activeConnectorId = ConnectorController.state.activeConnector?.id
```

Comment:
For multichain functionality it's better to get the given `namespace`'s connector id before active namespace. That'd be something like 
```suggestion
    const activeConnectorId = ConnectorController.getConnectorId(namespace) || ConnectorController.state.activeConnector?.id
```

Reply:
I realized that ConnectorControler does not guarantee that it's connected. Using connectedConnectorId from storage instead,. And yes, checking via namespace 🫡

Reply:
yessir, check the code now!

Reply:
Seems still same, what has changed? 🤔